src/components/i18n/FormattedCompMessage.js

```diff
@@ -0,0 +1,187 @@
+// @flow
+import * as React from 'react';
+import { injectIntl } from 'react-intl';
+import _ from 'lodash';
```

Comment:
Please avoid importing the root lodash module, as much of it is unused and will increase our bundle size. You should be able to do `import isNaN from 'lodash/isnan';`